.github/workflows/daily-checks.yml

```diff
@@ -0,0 +1,14 @@
+name: Cron Daily Checks
```

Comment:
I created a generic name as I think we can fit many of these checks in there. Like license headers check, security checks...